testsuite/forge-cli/src/main.rs

```diff
@@ -141,9 +141,9 @@ struct Resize {
     move_modules_dir: Option<String>,
     #[structopt(
         long,
-        help = "If set, uses kubectl port-forward instead of assuming k8s DNS access"
+        help = "If set, dont use kubectl port forward to access the cluster"
     )]
-    port_forward: bool,
+    connect_directly: bool,
```

Comment:
You can also just change the default to true, especially if it's clap.

Reply:
Default = true raises a warning that this has no effect, which is why i invert the arg because i think inverting it makes more sense in this case

Reply:
Although possibly --no-port-forward is actually more intuitive

Reply:
Interesting, I'm pretty sure the default =true works in clap